src/com/adgoji/mollie/client.clj

```diff
@@ -83,13 +83,13 @@
        (cond-> body
          :always         (response-transformer)
          check-response? (spec/check spec))
-       (let [anomaly (case status
-                       (400 401 422) ::anomalies/incorrect
-                       403           ::anomalies/forbidden
-                       (404 410)     ::anomalies/not-found
-                       405           ::anomalies/unsupported
-                       500           ::anomalies/fault
-                       503           ::anomalies/busy)
+       (let [anomaly (case (long status)
```

Comment:
Can't wrap my head around why we need long here.

Reply:
To avoid performance warning:

<img width="879" alt="image" src="https://github.com/AdGoji/mollie/assets/100711682/f4d6ba31-2362-4a22-973a-c8ff0095c24b">